src/node_options.cc

```diff
@@ -281,6 +281,11 @@ DebugOptionsParser::DebugOptionsParser() {
 }
 
 EnvironmentOptionsParser::EnvironmentOptionsParser() {
+  AddOption("--conditions",
+            "set the conditional exports conditions",
+            &EnvironmentOptions::conditions,
+            kAllowedInEnvironment);
+  AddAlias("-cc", "--conditions");
```

Comment:
we shouldn't use a multi-character `-` flag if it doesn't split up into component parts of equivalent things per standard *nix guidelines. Maybe we can find an alias later.

Reply:
Ok I've removed the alias for now. Would you be ok with using another letter like `-m`?

Reply:
sure, any letter

Reply:
Ok I've pushed up the `-m` alias for now, we can discuss further though.